scripts/main.js

```diff
@@ -0,0 +1,166 @@
+    title: 'Maru Sisay',
+    subTitle: 'Full stack developer fro more than 10 yeats with react JS',
+    content: 'Ryan had been leading open-source projects at Renewable Energy.',
+  },
+];
+
+// attache hide mobile menu EVENT
+const EventHideMobileMenu = (element, event) => {
+  element.addEventListener(event, () => {
+    navigation.setAttribute('class', 'hide');
+  });
+};
+
+// attach show mobile menu EVENT to any element with any event(click,hover,more...).
+const EventShowMobileMenu = (element, event) => {
+  element.addEventListener(event, () => {
+    navigation.setAttribute('class', '');
+  });
+  const menuElements = Array.from(navigation.querySelectorAll('li'));
+  menuElements.push(...Array.from(navigation.querySelectorAll('#close-menu')));
+  menuElements.push(...Array.from(navigation.querySelectorAll('.remaining')));
+  for (let i = 0; i < menuElements.length; i += 1) {
+    EventHideMobileMenu(menuElements[i], 'click');
+  }
+};
+
+// create list of reputaion's (speakers)
+const createSpeakerCard = (speaker) => {
+  const reputation = document.createElement('div');
+  reputation.classList.add('reputation');
+
+  const witnessImgContainer = document.createElement('span');
+  witnessImgContainer.classList.add('witness-img');
+  const witnessImg = document.createElement('img');
+  witnessImg.src = speaker.witnessImage;
+  witnessImg.alt = speaker.imageAlt;
+  witnessImgContainer.appendChild(witnessImg);
+  reputation.appendChild(witnessImgContainer);
+
+  const text = document.createElement('span');
+  text.classList.add('text');
+
+  const title = document.createElement('h2');
+  title.classList.add('title');
+  title.textContent = speaker.title;
+  text.appendChild(title);
+
+  const subTitle = document.createElement('p');
+  subTitle.classList.add('sub-title');
+  subTitle.textContent = speaker.subTitle;
+  text.appendChild(subTitle);
+
+  const hr = document.createElement('hr');
+  hr.classList.add('left-tick');
+  text.appendChild(hr);
+
+  const content = document.createElement('p');
+  content.classList.add('content');
+  content.textContent = speaker.content;
+  text.appendChild(content);
+  reputation.appendChild(text);
+  return reputation;
+};
+
+const updateSpeakers = (speakerSection, start = 0, end = speakers.length) => {
+  for (let i = start; i < end; i += 1) {
+    speakerSection.appendChild(createSpeakerCard(speakers[i]));
+  }
+};
+const removeChildren = (element, start = 0, end = speakers.length) => {
+  const children = Array.from(element.childNodes);
+  for (let i = start; i < end && children[i]; i += 1) {
+    element.removeChild(children[i]);
+  }
+};
+
+// reload window on screen break point changes to keep the dom clean
+const winWidth = window.innerWidth;
+window.addEventListener('resize', () => {
+  if ((winWidth < 768 && window.innerWidth > 768)
+    || (winWidth > 768 && window.innerWidth < 768)) {
+    window.location.reload();
+  }
+});
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Function creates HTML elements with the data from each object when DOMContent is loaded 😎

Reply:
@mateo951 Thank you.